haystack/pipeline.py

```diff
@@ -832,3 +837,192 @@ def run(self, **kwargs):
             documents = documents[: self.top_k_join]
         output = {"query": inputs[0]["query"], "documents": documents, "labels": inputs[0].get("labels", None)}
         return output, "output_1"
+
+
+class RayPipeline(Pipeline):
+    def __init__(self, pipeline_type: str = "Query"):
```

Comment:
We should allow Ray init params as well to customize ray ie ditched mode, ip-port etc.

Reply:
@lalitpagaria yes absolutely, that's the plan. We need the parameters to be able to connect to a remote Ray cluster.